src/schematics/deploy/functions-templates.ts

```diff
@@ -0,0 +1,29 @@
+export const defaultPackage = `{
+  "name": "functions",
+  "description": "Angular Universal Application",
+  "scripts": {
+    "lint": "",
+    "serve": "firebase serve --only functions",
+    "shell": "firebase functions:shell",
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "8"
```

Comment:
It's beta, but what do you think of defaulting to Node 10? I think it would be a better experience out of box.

Reply:
Let's warn/error if the engine version doesn't match their local development.